bigquery_etl/stripe/allowed_fields.py

```diff
@@ -33,11 +33,24 @@ def _valid_float(value: str):
 
 
 def get_rooted_schema(type_) -> bigquery.SchemaField:
-    """Load schema for given stripe type from json."""
+    """Load schema for given stripe type from json.
+
+    If the schema file does not exist, return the schema for data.{type_} from the event
+    schema.
```

Comment:
instead of duplicating the resource's schema between event.schema.json and its own file.